extensions/oidc-common/runtime/src/main/java/io/quarkus/oidc/common/runtime/OidcCommonConfig.java

```diff
@@ -307,14 +300,15 @@ public static class Jwt {
             public Map<String, String> claims = new HashMap<>();
 
             /**
-             * Signature algorithm, also used for the {@link #keyFile} property.
-             * Supported values: RS256, RS384, RS512, PS256, PS384, PS512, ES256, ES384, ES512, HS256, HS384, HS512.
+             * Optional. The signature algorithm, also used for the {@link #keyFile} property.
```

Comment:
`RS256` is the default but see, it only works when `keyFile` is set

Reply:
Updated. Thanks.